eng/common/scripts/Update-DocsMsToc.ps1

```diff
@@ -266,12 +268,12 @@ if ($otherPackages) {
 
```

Comment:
One comment above is referring to GetClientPackageNode you should update it to match the new name.